examples/model_selection/plot_det.py

```diff
@@ -114,3 +119,29 @@
 # DET curves give direct feedback of the detection error tradeoff to aid in
 # operating point analysis. The user can then decide the FNR they are willing to
 # accept at the expense of the FPR (or vice-versa).
+#
+# Chance level of the ROC and DET curves
```

Comment:
I don't understand what you mean by "chance level".

Reply:
Chance level is in general not easy to define, as it depends on what is the model considered as representative of pure chance. In this case, it is a `DummyClassifier` with the default "prior" strategy. Should I add such specification to the text below?

Reply:
Maybe we could use a more specific expression such as "constant prediction baseline" instead of chance level?

Reply:
> Maybe we could use a more specific expression such as "constant prediction baseline" instead of chance level?

I prefer the wording "chance level" for consistency with the `plot_chance_level` API (see e.g. https://github.com/scikit-learn/scikit-learn/pull/25987), even if suboptimal.

Reply:
Shall we add the `plot_chance_level` option to `DetCurveDisplay` or do that in a follow-up PR to keep the 2 sub-problems independently mergeable?

Reply:
I would leave it for a follow-up PR, yes.

Reply:
I just want to comment that I actually also find this terminology confusing (and simply wrong?) since the baseline has nothing to do with 'chance'. (Same in the PR case, incidentally.)